Server/Controllers/UserController.js

```diff
@@ -0,0 +1,68 @@
+const User = require("../Models/UserModel");
+
+const post_LoginUser = async (req, res) => {
```

Comment:
@Ayo-Awe  hey do you feel using a session is better?  

next time you can consider using jwt authentication so it will be easier for you and the frontend. 

nice work by the way

Reply:
Thanks 🙏, I look into it.